.devcontainer/docker-compose.yml

```diff
@@ -23,7 +23,11 @@ services:
     environment:
       PGADMIN_DEFAULT_EMAIL: ${PGADMIN_DEFAULT_EMAIL:-pgadmin4@pgadmin.org}
       PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_DEFAULT_PASSWORD:-admin}
+      PGADMIN_LISTEN_PORT: 5050
+    volumes:
+      - ./servers.json:/pgadmin4/servers.json
+      - ./pgadmin_files:/var/lib/pgadmin
     ports:
-      - 5050:80
+      - 5050:5050
```

Comment:
I would not change that (see previous comment)

Reply:
If you get it working without that, please let me know.

Port 80 tends to be a beast...

Reply:
You were actually right. It worked without a change.